PartsApp/DatabaseHelper/DbStructureCheck.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Infrastructure;
+using System;
+using System.Data.SQLite;
+
+namespace PartsApp.DatabaseHelper
+{
+    internal static class DbStructureCheck
+    {
+        internal static void EnsureNewDBColumnsExists()
+        {
+            using (var connection = DbConnectionHelper.GetDatabaseConnection() as SQLiteConnection)
+            {
+                connection.Open();
+                EnsureStorageCellColumnExistsInSparePartsDB(new SQLiteCommand("", connection));            
```

Comment:
Why pass a dummy CMD object as a parameter? And conventionally also this is incorrect: typically we pass an open connection

Reply:
corrected